src/util.js

```diff
@@ -1,5 +1,5 @@
 export const valueExistInSelected = (value, values, props) =>
-  !!values.find((val) => getByPath(val, props['valueField']) === value);
+  !!values.find((val) => getByPath(val, props['valueField']) === value || val.label === value);
```

Comment:
Just a question: shouldn't the access to `val.label` by protected with `getByPath` and using `props['labelField']` too?

Reply:
@DLatheron you right. It should, will fix and release tonight.